veles/secrets/recaptchakey/detector.go

```diff
@@ -0,0 +1,95 @@
+package recaptchakey
+
+import (
+	"bufio"
+	"bytes"
+	"regexp"
+	"slices"
+
+	"github.com/google/osv-scalibr/veles"
+)
+
+var (
+	// inlinePattern matches an inline assignment of a captcha secret key and captures its value (works for .env .json and .yaml)
+	//
+	// { and } are excluded to not overlap with inline json
+	inlinePattern = regexp.MustCompile(`(?i)captcha[^\{\}\n\r]{0,50}(?:secret|private)[^\{\}\n\r]{0,50}(6[A-Za-z0-9_\-'"]{39})\b`)
```

Comment:
I'd add a \b before the `6[A-Z...` blob as well - since otherwise the `[^\{\}\n\r]` before it could make us find long unrelated blobs that have 6 in them somewhere